include/mbedtls/aes.h

```diff
@@ -170,8 +171,9 @@ int mbedtls_aes_setkey_enc( mbedtls_aes_context *ctx, const unsigned char *key,
 /**
  * \brief          This function sets the decryption key.
  *
- * \param ctx      The AES context to which the key should be bound.
- * \param key      The decryption key.
+ * \param ctx      The AES context to which the key should be bound. Must not
```

Comment:
As commented in #1643, I have changed my mind on that in the meantime - I think we should formulate preconditions more precisely than just saying that the context must not be `\c NULL`, but instead say that it must be initialized.

Reply:
Fixed.